plugins/modules/elb_classic_lb.py

```diff
@@ -744,7 +744,7 @@ def __init__(self, module):
         if security_group_names:
             # Use the subnets attached to the VPC to find which VPC we're in and
             # limit the search
-            if self.elb.get('Subnets'):
+            if self.elb.get('Subnets', None):
```

Comment:
Fails if ELB doesn't exsist (even if called with `state: absent`)
`elb` is undefined. `get()` throws error. 

Replace with:
`if self.elb is not None and self.elb.get('Subnets', None):`